data_safe_haven/commands/admin.py

```diff
@@ -5,11 +5,13 @@
 
 import typer
 
-from .admin_add_users import admin_add_users
-from .admin_list_users import admin_list_users
-from .admin_register_users import admin_register_users
-from .admin_remove_users import admin_remove_users
-from .admin_unregister_users import admin_unregister_users
+from data_safe_haven.administration.users import UserHandler
+from data_safe_haven.config import Config, DSHPulumiConfig
+from data_safe_haven.context import ContextSettings
+from data_safe_haven.exceptions import DataSafeHavenError
+from data_safe_haven.external import GraphApi
+from data_safe_haven.functions import sanitise_sre_name
+from data_safe_haven.utility import LoggingSingleton
 
 admin_command_group = typer.Typer()
```

Comment:
All the `admin` commands are to do with user management. Do we envisage future commands that do other things or should we simplify e.g. `dsh admin list-users` to `dsh users list`?

Reply:
Let's move that to `users`  then. In the future there might be `dsh admin users ...` and `dsh admin something-else ...`

Reply:
Done in 96f91b76